### PR TITLE
add ending message

### PR DESCRIPTION
**Pull request by ddokupil**
Add ending message to indicate script execution properly finished

### CODE REVIEW

**Comment by ddokupil** [comment]
My business need is to add a message when script reaches its end. We need it in our internal nightly. Right now we cannot sense whether it prematurely finished or just failed some unit tests.

**Review by luotao1 — COMMENTED** [review]
@bingyanghuang You can use `test=document_fix` to test this PR only ON CPU.

File: paddle/scripts/paddle_build.sh [hunk 1]
```diff
@@ -1099,6 +1099,7 @@ function main() {
         exit 1
         ;;
       esac
+      echo "paddle_build script finished as expected"
```
Comment (luotao1) [inline paddle/scripts/paddle_build.sh:1102]: paddle_build script finished as expected.

**Comment by ddokupil** [comment]
test=develop test=document_fix

**Review by luotao1 — APPROVED** [review]
LGTM

**Pull request merged**